codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2ConnectionDecoder.java

```diff
@@ -135,6 +153,59 @@ public Http2Settings localSettings() {
         return settings;
     }
 
+    /**
+     * Applies settings sent from the local endpoint.
+     */
+    private void applyLocalSettings(Http2Settings settings) throws Http2Exception {
```

Comment:
Nothing has changed here. This method was previously inside the scope of the inner class `FrameReadListener`, but now it is being called from both the inner class and the outer class and that is why it was moved.